mreg_cli/commands/host_submodules/core.py

```diff
@@ -140,34 +141,81 @@ def add(args: argparse.Namespace) -> None:
             metavar="NAME/IP",
         ),
         Flag("-force", action="store_true", description="Enable force."),
+        Flag(
+            "-override",
+            short_desc="Comma separated override list, requires -force.",
+            description=(
+                "Comma separated overrides for forced removal. Requires -force."
+                "Accepted overrides: 'cname', 'ipadresses', 'mxs', 'srv', 'ptr', 'naptr'."
+                "Example usage: '-override cnames,ipaddresses,mxs'"
```

Comment:
Should overrides all be in singular form to match the DNS record types? I.e. `cname` instead of `cnames`. Granted, ipaddress is not a record type, but that shouldn't be too confusing in this context.

Reply:
```suggestion
                "Accepted overrides: 'cname', 'ipadress', 'mx', 'srv', 'ptr', 'naptr'."
                "Example usage: '-override cname,ipaddress,mx'"
```